src/authfile-update

```diff
@@ -37,36 +38,91 @@ if [[ $# -ne 1 ]]; then
 fi
 
 TOPOLOGY=${TOPOLOGY:-https://topology.opensciencegrid.org}
+CACHE_FQDN=${CACHE_FQDN:-$(hostname -f)}
+ORIGIN_FQDN=${ORIGIN_FQDN:-$(hostname -f)}
 
-if [[ $1 == --cache ]]; then
-    CACHE_FQDN=${CACHE_FQDN:-$(hostname -f)}
-    ret=0
-    fetch "${TOPOLOGY}/stashcache/authfile?cache_fqdn=${CACHE_FQDN}"        /run/stash-cache-auth/Authfile
-    ret=$(( $ret | $? ))
-    fetch "${TOPOLOGY}/stashcache/authfile-public?cache_fqdn=${CACHE_FQDN}" /run/stash-cache/Authfile
-    ret=$(( $ret | $? ))
-    fetch "${TOPOLOGY}/stashcache/scitokens?cache_fqdn=${CACHE_FQDN}"        /run/stash-cache-auth/scitokens.conf
+DESTDIR=${DESTDIR:-/run}
+
+
+append_authfile_local () {
+    local base="$1"
+    if [[ -s $base && -f ${base}.local ]]; then
+        echo -e "\n# The following lines are from $(basename "${base}.local"):" >> "$base"
+        cat "${base}.local" >> "$base"
+    fi
```

Comment:
Just checking:

So if $base is not empty then append ${base}.local to $base,

but if $base is empty or does not exist, then do not create $base or append ${base}.local to it?

Reply:
Right, `$base` is supposed to be downloaded from topology; if it's empty or missing then that means there was an error with either the data or with the download and we shouldn't create a partial authfile from just `${base}.local`.

Reply:
ok then